wallet/lib/services/repository/repository.dart

```
@@ -1859,7 +1859,7 @@ class RepositoryImp implements Repository {
     } on Failure catch (_) {
       return Left(_);
     } on Exception catch (_) {
-      recordErrorInCrashlytics(_);
+      // recordErrorInCrashlytics(_);
```

Should there be dead code here?

---

Yes i silenced error. Until chain starts receiving custom events.